### PR TITLE
refine readme

### PR DESCRIPTION
**Pull request by cyyeh**
- update instructions for running wren-ai-service for development
- simplify demo app setup
- update Poetry version
  - readme
  - ci
- minor performance improvement(reference: https://www.youtube.com/watch?v=7jtzjovKQ8A)
  - uvicorn
    - explicitly use `uvloop`
    - explicitly use `httptools`
  - fastapi
    - use `ORJSONResponse` instead of `JSONResponse`

### CODE REVIEW

File: wren-ai-service/demo/Makefile
Comment (paopa) [inline wren-ai-service/demo/Makefile:1]: Would you be able to move the run command to the primary make file? Almost all commands here are the same in the primary make file. thus, I think we can consider moving the one to start the AI demo-site to there and delete this file.

**Review by paopa — COMMENTED** [review]
Just curious do we have the chart or table to compare the original performance and this one with Uvloop and Orjson response

**Comment by cyyeh** [comment]
> Just curious do we have the chart or table to compare the original performance and this one with Uvloop and Orjson response

by default uvicorn used "uvloop" and "httptools"; however, I just want to make it more explicit here :)

for orjson, it's not easier to see the difference, since the payload we are using is not big enough; however, I think we still should use orjson if it's already available :)

**Review by paopa — APPROVED** [review]
LGTM. thanks for your work! after the CI done, we could merge this PR.

**Pull request merged**